### PR TITLE
MINOR: Add fixes for adding new dsl naming page

### PR DESCRIPTION
**Pull request by bbejeck**
We needed to add the appropriate re-directs for the new topology naming page.  I've tested the links by rendering the site locally.

### CODE REVIEW

**Comment by bbejeck** [comment]
ping @mjsax @ableegoldman @londoncalling

**Comment by mjsax** [comment]
@bbejeck Is there a PR again AK repo, too?

**Review by mjsax — APPROVED** [review]

File: documentation/streams/developer-guide/dsl-topology-naming.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<!-- should always link the the latest release's documentation -->
```
Comment (ableegoldman) [inline documentation/streams/developer-guide/dsl-topology-naming.html]: nit: "the" is repeated, but that's not really a big deal (and this is presumably the case for all the other pages too so...)

**Comment by bbejeck** [comment]
> Is there a PR again AK repo, too?

Yes - https://github.com/apache/kafka/pull/7899

File: documentation/streams/developer-guide/dsl-topology-naming.html (hunk 1 above)
Comment (bbejeck) [reply documentation/streams/developer-guide/dsl-topology-naming.html]: good catch!

**Review by bbejeck — COMMENTED** [review]
@ableegoldman I've updated this to remove all redundant `the` in the redirect comments.

**Comment by bbejeck** [comment]
Merged #248 into asf-site

**Pull request merged**